src/calls.rs

```diff
@@ -137,7 +137,7 @@ pub extern "C" fn sudo(
 ) -> UnmanagedVector {
     call_2_args(
         // TODO: rename to sudo when change in CosmWasm
```

Comment:
```suggestion
```

Reply:
Good eye, removed while fixing the broken rust test